zp-relayer/router.ts

```diff
@@ -29,6 +31,13 @@ router.use((err: any, req: Request, res: Response, next: NextFunction) => {
   next()
 })
 
+router.use((req: Request, res: Response, next: NextFunction) => {
+  if (config.requireTraceId && req.headers[TRACE_ID]) {
+    logger.info('TraceId %s %s', req.headers[TRACE_ID], req.path)
```

Comment:
Let's put traceId and request path both under respective json keys in the log.

Http request from users are currently logged to a separate file, maybe it makes sense to put traceId there and log these messages to console as well.

Reply:
>Let's put traceId and request path both under respective json keys in the log.

Agree

> maybe it makes sense to put traceId there and log these messages to console as well.

`zp.log` where every request is dumped, already includes all headers, so trace id will be saved there as well.
I don't think logging everything to console that is saved in `zp.log` is a good idea, as logs will be mostly flooded with such request logs which are >500 characters each

Reply:
> I don't think logging everything to console that is saved in zp.log is a good idea, as logs will be mostly flooded with such request logs which are >500 characters each

I think we do not need all of these 500 characters, but ok, let's leave it for future discussion/PR anyway.